imagestreams/ruby-centos7.json

```diff
@@ -92,6 +92,23 @@
           "kind": "DockerImage",
           "name": "docker.io/centos/ruby-24-centos7:latest"
         }
+      },
+      {
+        "name": "2.5",
+        "annotations": {
+          "openshift.io/display-name": "Ruby 2.5",
+          "openshift.io/provider-display-name": "Red Hat, Inc.",
+          "description": "Build and run Ruby 2.5 applications on CentOS 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/s2i-ruby-container/blob/master/2.5/README.md.",
+          "iconClass": "icon-ruby",
+          "tags": "builder,ruby",
+          "supports": "ruby:2.5,ruby",
+          "version": "2.5",
+          "sampleRepo": "https://github.com/openshift/ruby-ex.git"
```

Comment:
This was now moved to `https://github.com/sclorg/ruby-ex.git`

Reply:
Should we rename all the sampleRepo for not only 2.5 but also older versions of Ruby?

Reply:
Yes, I think so, but that is out of the scope of this PR.

Reply:
@pvalena I only fixed it for Ruby 2.5.
I want ask you to modify other parts later if you have a time.